docs/source/guides/using_dask_entitysets.ipynb

```diff
@@ -0,0 +1,245 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49c496cb",
+   "metadata": {},
+   "source": [
+    "Creating a feature matrix from a very large dataset can be problematic if the underlying pandas dataframes that make up the EntitySet cannot easily fit in memory. To help get around this issue, Featuretools supports creating `EntitySet` objects from Dask dataframes. A Dask `EntitySet` can then be passed to `featuretools.dfs` or `featuretools.calculate_feature_matrix` to create a feature matrix, which will be returned as a Dask dataframe. In addition to working on larger than memory datasets, this approach also allows users to take advantage of the parallel and distributed processing capabilities offered by Dask.\n",
+    "\n",
+    "This guide will provide an overview of how to create a Dask `EntitySet` and then generate a feature matrix from it. If you are already familiar with creating a feature matrix starting from pandas DataFrames, this process will seem quite familiar, as there are no differences in the process. There are, however, some limitations when using Dask dataframes, and those limitations are reviewed in more detail below.\n",
+    "\n",
+    "## Creating EntitySets\n",
+    "\n",
+    "For this example, we will create a very small pandas DataFrame and then convert this into a Dask DataFrame to use in the remainder of the process. Normally when using Dask, you would just read your data directly into a Dask DataFrame without the intermediate step of using pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96a8f65a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import featuretools as ft\n",
+    "import pandas as pd\n",
+    "import dask.dataframe as dd\n",
+    "id = [0, 1, 2, 3, 4]\n",
+    "values = [12, -35, 14, 103, -51]\n",
+    "df = pd.DataFrame({\"id\": id, \"values\": values})\n",
+    "dask_df = dd.from_pandas(df, npartitions=2)\n",
+    "dask_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0c3d410",
+   "metadata": {},
+   "source": [
+    "Now that we have our Dask DataFrame, we can start to create the `EntitySet`. Inferring logical types for the columns in a Dask dataframe can be computationally expensive. To avoid this expense, logical type inference can be skipped by supplying a dictionary of logical types using the `logical_types` parameter when calling `es.add_dataframe()`. Logical types can be specified as Woodwork LogicalType classes, or their equivalent string representation.\n",
```

Comment:
Not sure if we need to link to Woodwork guides _every_ time we reference Woodwork types, but so far that's pretty much what I've been doing. If we want to link, we can link to the featuretools-woodwork guide.

Reply:
Another woodwork thing we could link to is the guide on Dask DataFrames in Woodwork (if we feel the need to mention that EntitySets can be built from dataframes that already have woodwork initialized in this section)

Reply:
Added a link